src/java.base/share/classes/sun/net/www/MimeTable.java

```diff
@@ -45,10 +45,10 @@ public class MimeTable implements FileNameMap {
     private static final int HASH_MARK = '#';
 
     /** Keyed by content type, returns MimeEntries */
-    private Hashtable<String, MimeEntry> entries = new Hashtable<>();
+    private final Hashtable<String, MimeEntry> entries = new Hashtable<>();
```

Comment:
Drive-by observation: These two `Hashtable`s can be changed to `HashMap` since every method that uses them is `synchronized`, making the implicit synchronization from `Hashtable` redundant.